doc/modules/clustering.rst

```diff
@@ -278,6 +285,67 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+
+:class:`KMedoids` is related to the :class:`KMeans` algorithm. While
+:class:`KMeans` tries to minimize the within cluster sum-of-squares,
+:class:`KMedoids` tries to minimize the sum-of-distances between points labeled
+as the medoids of the cluster an all points assigned to that cluster.
+Furthermore, the distance metric used by :class:`KMedoids` can be freely chosen
+while :class:`KMeans` uses a fixed Euclidean distance.
+
+:class:`KMedoids` can be more robust to noise and outliers than :class:`KMeans`
+as e.g. it will choose one of the cluster members as the medoid while
+:class:`KMeans` will move the center of the cluster towards the outlier which
+might in turn move other points away from the cluster centre.
+
+Please note that K-Medoids is different from the K-Medians algorithm as it
+always chooses a point from the dataset as the cluster centre. K-Medians might
+choose a point which is not in the original dataset. For example, given the
+vectors (0,1), (1,0) and (2,2), the Manhattan-distance median is (1,1),
+which does not exist in the original data, and thus cannot be a medoid.
+
+.. topic:: Examples:
+
+ * :ref:`example_cluster_plot_kmedoids_digits.py`: Applying K-Medoids on digits
+ with various distance metrics.
+
+**Algorithm description:**
+:class:`KMedoids` fitting algorithms have different variants, one of them being
```

Comment:
"There are several algorithms to compute K-Medoids, though KMediods currently only supports Partitioning Around Medoids (PAM)"